### PR TITLE
Small adjustments in scm and search in workspace css

### PR DESCRIPTION
**Pull request by jbicker**
#### What it does
Aligns header/input area of SCM and Search In Workspace Extensions.
Fixes https://github.com/theia-ide/theia/issues/5751

#### How to test
Start theia and switch between the sidebars with SCM and Search In Workspace 

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by kittaakos** [comment]
Nice 👍 It is much better than it was.

The SIW `input` seems to have a greater `height` than the `input` in SCM.

screencast 2019-09-05 15-55-58

File: packages/scm/src/browser/style/index.css [hunk 1]
```diff
@@ -100,7 +100,8 @@
     box-sizing: border-box;
     height: 26px;
     min-height: 26px;
-    padding: 4px 4px 4px 4px;
+    padding: 4px;
```
[the remark this answers is not in the document]
Comment (jbicker) [reply packages/scm/src/browser/style/index.css]: Generally yes. In this case it is a single value for this textfield. Do you want a variable for every value?

**Comment by jbicker** [comment]
Adjusted the heights once more.

File: packages/scm/src/browser/style/index.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (kittaakos) [reply packages/scm/src/browser/style/index.css]: I thought `padding: 4px 4px 4px 4px;` and `padding: 4px;` are the same. 😊 Why the change?

[the remark this answers is not in the document]
Comment (akosyakov) [reply packages/scm/src/browser/style/index.css]: ok, i thought use 4px in scm and search to have the same offset

[the remark this answers is not in the document]
Comment (jbicker) [reply packages/scm/src/browser/style/index.css]: > I thought `padding: 4px 4px 4px 4px;` and `padding: 4px;` are the same. 😊 Why the change?

I like it short and to the point. :-)

**Pull request merged**